examples/secure_metricbeats.md

```diff
@@ -0,0 +1,42 @@
+# Secure Metricbeats
```

Comment:
Shouldn't it be Metricbeat?

<img width="1151" alt="image" src="https://github.com/user-attachments/assets/bcb4e3d4-98e7-4106-bf37-8c56668bf73c">